src/trayicon.cpp

```diff
@@ -610,23 +622,24 @@ void TrayIcon::startThunderbird()
     connect( mThunderbirdProcess, &QProcess::errorOccurred, this, &TrayIcon::tbProcessError );
 #endif
 
-    mThunderbirdProcess->start(executable, arguments);
+    mThunderbirdProcess->start(executable, args);
 }
 
 void TrayIcon::tbProcessError(QProcess::ProcessError )
 {
 #ifdef Q_OS_WIN
-    if (mThunderbirdUpdaterProcess->attach() == AttachResult::SUCCESS) {
+    if (mThunderbirdUpdaterProcess->attach() == AttachResult::SUCCESS)
+    {
```

Comment:
I don't think you wanted to include that.

Reply:
That's windows, right?

Reply:
I generally use the next-line brackets, so the functions I modify I do the changes :)

Reply:
If you can find the time, I would really appreciate it if you add a `.clang-format` file or similar to the project. That way we can both use the same code style for this project.